src/components/Request/Pay/Pay.tsx

```diff
@@ -23,18 +24,29 @@ export const PayRequestLink = () => {
 
     const fetchPointsEstimation = async (
         requestLinkDetails: { recipientAddress: any; chainId: any; tokenAmount: any },
-        tokenPrice: { price: number; chainId: string; decimals: any } | undefined
+        tokenPriceData: ITokenPriceData | undefined
     ) => {
         const estimatedPoints = await estimatePoints({
             address: requestLinkDetails.recipientAddress,
             chainId: requestLinkDetails.chainId,
-            amountUSD: Number(requestLinkDetails.tokenAmount) * (tokenPrice?.price ?? 0),
+            amountUSD: Number(requestLinkDetails.tokenAmount) * (tokenPriceData?.price ?? 0),
             actionType: ActionType.CLAIM, // When API on prod will be ready lets change it to ActionType.FULFILL
         })
 
         setEstimatedPoints(estimatedPoints)
     }
 
+    const fetchRecipientAddress = async (address: string): Promise<string> => {
+        if (!address.endsWith('eth')) {
+            return address
+        }
+        const resolvedAddress = await utils.resolveFromEnsName(address.toLowerCase())
+        if (!resolvedAddress) {
+            throw new Error('Failed to resolve ENS name')
+        }
+        return resolvedAddress
+    }
```

Comment:
ignore .xyz and .luxe, wtf is that

But a potentially bigger issue @jjramirezn: when we create a request link, we now store jota.eth as the request address. That seems to be a potential issue for:
- localStorage tx history
- maybe backend db?

wdyt? Problematic or no?

Reply:
Hmmm, the issue lies if something that expect a hex address gets an ens. I think it's good to support ens on request link so I don't see a issue with storing and respecting the ens.

Something that can be done is storing the address to which the payment was done (the resolved one) that way we don't lose that information (we have the tx hash so the info is not lost-lost, but it could be useful)